scripts/functions.py

```diff
@@ -106,4 +107,404 @@ def daylightSavings(inData):
+    # INPUT:   countries = Dictionary holding countries as the key and subregion as the values in a list
+    #          techsMaster = List of the technologies to print over for power techs
+    #          rnwTechs = List of the technologies to print over for renewable techs
+    #          mineTechs = List of the technologies to print over for mining techs
+    #          stoTechs = List of the technologies to print over for storage techs
+    # OUTPUT:  data = Merged tech lists
+    # get power generator technology list 
+    data = []
+
+    for i in range(len(techsMaster)):
+        data.append(['PWR',techsMaster[i]])
+
+    for i in range(len(rnwTechs)):
+        data.append(['RNW',rnwTechs[i]])
+
+    for i in range(len(mineTechs)):
+        data.append(['MIN',mineTechs[i]])
+
+    for i in range(len(stoTechs)):
+        data.append(['STO',stoTechs[i]])
+    
+    return data
+
+def initializeCanadaUsaModelParameters(topLevelRegion):
+    # PURPOSE: Initializes necessary parameters for config.py and UsaData.py, which create
+    # Tech and Fuel sets for Canada and the US, respectively
+    # INPUT:   topLevelRegion = String indicating region: 'CAN' or 'USA'
+    # OUTPUT:  years = List from 2019 to 2051
+    #          regions = List for regions
+    #          emissions = List for CO2 emissions
+    #          techsMaster = List for power technologies
+    #          rnwTechs = List for renewable technologies
+    #          mineTechs = List for mining technologies
+    #          stoTechs = List for storage technologies
+    #          countries = Dictionary for holding countries as the key and subregion as the values in a list
+    # get power generator technology list 
+
+    # Regions in the model
+    regions = ['NAmerica']
+
+    # Years to loop over 
+    years = list(range(2019,2051,1))
+
+    # Emission Types
+    emissions = ['CO2']
+
+    # Storages
+    # storages = [] 
+
+    # PWR Technologies
+    techsMaster = [
+        'BIO', # Biomass
+        'CCG', # Gas Combind Cycle 
+        'CTG', # Gas Combustion Turbine
+        'COA', # Coal
+        'COC', # Coal CCS
+        'HYD', # Hydro 
+        'SPV', # Solar
+        'URN', # Nuclear
+        'WND', # Wind
+        #'P2G', # Power to Gas
+        #'FCL'  # Fuel Cell
+    ]
+
+    #RWN Technologies
+    rnwTechs = [
+        'HYD', # Hydro 
+        'SPV', # Solar
+        'WND', # Wind
+        'BIO', # Biomass
+    ]
+
+    #MIN TEchnologies
+    mineTechs = [
+        'COA', # Coal
+        'GAS', # Gas
+        'URN', # Nuclear
+    ]
+
+    #STO Technologies
+    stoTechs = [
+        #'TNK' #Tank
+    ]
+
+    #Countries and subregions in the model
+    countries = {
+        topLevelRegion:[]
+        }
+
+    #Read in subregions. This step is needed cause other scripts use the 
+    #provincial breakdown by subregion in the excel file 
+    sourceFile = '../dataSources/Regionalization.xlsx'
+    for country in countries:

```

Comment:
Do we every have more then one country? Seems like country will always either just be CAN or USA?